src/Nethermind/Nethermind.Core/Collections/StackList.cs

```diff
@@ -1,15 +1,17 @@
 // SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
 // SPDX-License-Identifier: LGPL-3.0-only
 
+using System;
 using System.Collections.Generic;
+using System.Runtime.InteropServices;
 
 namespace Nethermind.Core.Collections
 {
-    public class StackList<T> : List<T>
+    public sealed class StackList : List<int>
```

Comment:
I know it is a bit of YAGNI, but I liked it being generic, can we achieve it without making it specialized?

Reply:
Can do, but its only used in a single place

Reply:
Made generic again